### PR TITLE
Add default to property specifier

### PR DESCRIPTION
**Pull request by xNekOIx**

### CODE REVIEW

**Review by SergeyKorochansky — APPROVED** [review]

File: templates/body_template.hbs [hunk 1]
```diff
@@ -16,17 +16,20 @@
...
         builderBlock(builder);
     }
     return [[self alloc] initWithBuilder:builder modelVersion:1];
 }
 
 - (instancetype)init
 {
-    {{name}}Builder *builder = [{{name}}Builder new];
-    return [[[self class] alloc] initWithBuilder:builder modelVersion:1];
+    return [self create:NULL];
```
Comment (SergeyKovalenko) [inline templates/body_template.hbs]: Here `create:`  it's a class method (static). So it will crash with `unrecognized selector sent to instance`

Comment (xNekOIx) [reply templates/body_template.hbs]: fixed

**Comment by xNekOIx** [comment]
@webzepter can we merge this already?

**Comment by SergeyKorochansky** [comment]
LGTM

**Pull request merged**